sample/src/androidTest/kotlin/com/agoda/sample/screen/SliderScreen.kt

```diff
@@ -0,0 +1,11 @@
+package com.agoda.sample.screen
+
+import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.slider.KSlider
+import com.agoda.kakao.text.KButton
+import com.agoda.sample.R
+
+class SliderScreen : Screen<SliderScreen>() {
+    val slider = KSlider { withId(R.id.slider) }
+    val buttonValue50 = KButton { withId(R.id.button_value_5) }
```

Comment:
R.id.button_value_**5** assigned for buttonValue**50**. Why?

Reply:
ahh, it's meant to be buttonValue5, used to be 50 but i changed it to use smaller numbers.